src/api/app/helpers/meta_tag_helper.rb

```diff
@@ -0,0 +1,13 @@
+module MetaTagHelper
+  def meta_title
+    content_for?(:meta_title) ? content_for(:meta_title) : 'Open Build Service'
```

Comment:
TIL, I didn't know `content_for?` existed. I looked at the source code to see what it does exactly.

So [`content_for` does this](https://github.com/rails/rails/blob/e69ff43060c1194d2a3bd9b8d9e23f3ae26b84b5/actionview/lib/action_view/helpers/capture_helper.rb#L162) when you only pass it a name:
```
@view_flow.get(name).presence
```

[`content_for?` checks](https://github.com/rails/rails/blob/e69ff43060c1194d2a3bd9b8d9e23f3ae26b84b5/actionview/lib/action_view/helpers/capture_helper.rb#L192) if it's present with:
```
@view_flow.get(name).present?
```

With this in mind, we can change this method's code to
```
content_for(:meta_title) || 'Open Build Service'
```

Same for `meta_image` below.

Reply:
Nice, this made things much cleaner!